### PR TITLE
Project: Virtual Private Cloud

### PR DESCRIPTION
**Pull request by zliang-akamai**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -100,3 +100,6 @@ require (
 )
 
 replace github.com/zclconf/go-cty => github.com/nywilken/go-cty v1.12.1 // added by packer-sdc fix as noted in github.com/hashicorp/packer-plugin-sdk/issues/187
+
+// Temporary replacement for linodego proj/vpc
+replace github.com/linode/linodego => github.com/linode/linodego v1.22.1-0.20230926172253-9eee757b17a6
```
Comment (lgarber-akamai) [inline go.mod:105]: Can we remove this replacement now that linodego has been bumped?

Comment (zliang-akamai) [reply go.mod:105]: Thanks, working on it.

**Review by lgarber-akamai — APPROVED** [review]
LGTM!

**Review by amisiorek-akamai — APPROVED** [review]
LGTM!

**Pull request merged**